packages/clerk-js/src/core/clerk.ts

```diff
@@ -255,6 +256,25 @@ export default class Clerk implements ClerkInterface {
 
   public isReady = (): boolean => this.#isReady;
 
+  get allowedRedirectOrigins(): (string | RegExp)[] | undefined {
+    if (!this.#options.allowedRedirectOrigins) {
+      const origins = [];
+      if (inBrowser()) {
+        origins.push(window.location.origin);
+        origins.push(window.location.origin + '/*');
+      }
+
+      const frontendApi = this.frontendApi;
+
+      origins.push(`https://*.${getETLDPlusOneFromFrontendApi(frontendApi)}`);
+      origins.push(`https://*.${getETLDPlusOneFromFrontendApi(frontendApi)}/*`);
```

Comment:
Why do we require the `/*` part here? Can you show an example where this is needed? Same question for line 264.

Reply:
Oh my bad this is not required, removing it as path doesn't matter for the check of same-origin.